docs/pages/cdp/cdp-Installing.md

```diff
@@ -24,40 +24,28 @@ toc: false
     sudo npm install -g @brightside/core
```

Comment:
Suggest removing `sudo` as the recommended approach is to setup your environment to install globals into your user directory.

Reply:
done.